### PR TITLE
Magento Chile Tests

### PR DESCRIPTION
**Pull request by cristopher-rodrigues**

### CODE REVIEW

File: magento/lib/pages/checkout.js [hunk 1]
```diff
@@ -391,6 +404,48 @@ export default class Checkout {
     });
   }
 
+  placeWithServiPag(data) {
+    validateSchema(CHECKOUT_SCHEMA.cl.servipag(), data, () => {
+      this[fillBilling](data);
+      this[fillDocument](data);
```
Comment (andercampanha) [inline magento/lib/pages/checkout.js]: There is no document field for this method.

File: magento/lib/pages/checkout.js [hunk 2]
```diff
@@ -391,6 +404,48 @@ export default class Checkout {
...
+      this.cy
+        .get('.via.webpay img.logoPT', { timeout: 15000 })
+        .should('be.visible');
+    });
+  }
+
+  placeWithMulticaja(data) {
+    validateSchema(CHECKOUT_SCHEMA.cl.multicaja(), data, () => {
+      this[fillBilling](data);
+      this[fillDocument](data);
```
Comment (andercampanha) [inline magento/lib/pages/checkout.js]: There is no document field for this method.

Comment (cristopher-rodrigues) [reply magento/lib/pages/checkout.js]: Yeah but this fill is optional and on schema is not allowed :)

File: magento/lib/pages/checkout.js (hunk 1 above)
Comment (cristopher-rodrigues) [reply magento/lib/pages/checkout.js]: Yeah but this fill is optional and on schema is not allowed :)

Comment (andercampanha) [reply magento/lib/pages/checkout.js]: Fair enough.

File: magento/lib/pages/checkout.js (hunk 2 above)
Comment (andercampanha) [reply magento/lib/pages/checkout.js]: Fair enough.

**Pull request merged**